server/migrations/20210203200024-add-title-to-chain-entities.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction((t) => {
+      return Promise.all([
+        queryInterface.addColumn(
+          'ChainEntities',
+          'title',
+          {
+            type: Sequelize.TEXT,
```

Comment:
Should be STRING (unless there's a good reason to use TEXT here?).

Reply:
I was just mirroring OffchainThread model! I'm happy with string. STRING defaults to a max length of 255, whereas TEXT is basically VarChar?

Reply:
Yep -- that's the distinction. I assume we'll be fine with titles maxing out at 255, but ofc if we want to support very long titles... then we can make it TEXT.